polus-infering-splinedist-plugin/src/utils.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import os 
+
+import numpy as np
+
+import bfio
+from bfio import BioReader
+
+from csbdeep.utils import normalize
+
+from splinedist.models import Config2D, SplineDist2D, SplineDistData2D
+from splinedist.utils import phi_generator, grid_generator
+
+import matplotlib
+matplotlib.rcParams["image.interpolation"] = None
+import matplotlib.pyplot as plt
+
+import filepattern
+from filepattern import FilePattern as fp
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("infer")
+logger.setLevel(logging.INFO)
+
+def create_plots(image,
+                 prediction,
+                 output_dir, 
+                 image_name):
+    
+    """ This function generates subplots of the 
+    original image with its predicted image.
+
+    Args:
+        input_len - the number of images in array_images and array_label
+        output_dir - the location where the jpeg files are saved
+        model - the neural network used to make the prediction
+    Returns:
+        None, saves images in output directory
+    """
+
+    fig, (a_image,a_prediction) = plt.subplots(1, 2, 
+                                               figsize=(8,5), 
+                                               gridspec_kw=dict(width_ratios=(1,1)))
+
+    plt_image = a_image.imshow(image)
+    a_image.set_title("Image")
+
+    plt_prediction = a_prediction.imshow(prediction)
+    a_prediction.set_title("Prediction")
+
+    plot_file = "{}.jpg".format(image_name)
+    plt.savefig(os.path.join(output_dir, plot_file))
+    plt.clf()
+    plt.cla()
+    plt.close(fig)
```

Comment:
As with the training function, this appears to be more of a utility to debugging. It looks like you're saving data as jpg files to the output directory which is invalid for WIPP, since all images should be saved a ome tiff.

I think this function should be removed and maybe moved to a test/debugging set of scripts.